### ISSUE TITLE
Attempt additional base64 decoding on cursed stamps

### ISSUE THREAD

**Issue by reinamora137** [stampchain-io/btc_stamps#264]
There are several where the decoding could be successful on cursed for instances where stamp_base64 is malformed

todo:

strip chars before and including a `,`  (some with base64,)
strip chars before and including a `:`  (some with stamp:)
strip invalid base64 chars

currently these decode to: application/octet-stream

this function was added in stamp.py to strip non base64, pre-checks will be needed to strip the others beforehand. 

```
#  NOTE: this can be used for cursed, but will change the txlist hash
def remove_non_base64_chars(base64_string):
    """
    Removes any non-base64 characters from the given base64 string.

    Args:
        base64_string (str): The base64 string to remove non-base64 characters from.

    Returns:
        str: The base64 string with non-base64 characters removed.
    """
    return ''.join(c for c in base64_string if c in base64.b64encode(base64.b64decode(base64_string)).decode())
```

**Comment by masteronchain** [comment]
For tokens where the description includes things like data:base64,... or file:base64,..., would those still be considered valid cursed Stamps?
Like, as long as we strip the prefix and decode the rest, is that acceptable?

**Comment by reinamora137** [comment]
> For tokens where the description includes things like data:base64,... or file:base64,..., would those still be considered valid cursed Stamps? Like, as long as we strip the prefix and decode the rest, is that acceptable?

Yes they should be cursed stamps now but the image may not render currently because of additional formatting outside the spec. This will resolve the formatting errors and ideally make the image render on some cursed stamps 

This won’t change any consensus rules or cause renumbering

**Comment by reinamora137** [comment]
not related to base64 decoding, but we are not re-processing text data to see if it's actually SVG. so we do have some cursed stamps that now render images properly which weren't previously. 

https://github.com/stampchain-io/btc_stamps/pull/609/commits
